package.json

```diff
@@ -48,20 +48,20 @@
 		"@sindresorhus/tsconfig": "^0.6.0",
 		"@types/benchmark": "^1.0.31",
 		"@types/node": "^12.12.7",
-		"@typescript-eslint/eslint-plugin": "^1.11.0",
-		"@typescript-eslint/parser": "^1.11.0",
+		"@typescript-eslint/eslint-plugin": "^2.25.0",
+		"@typescript-eslint/parser": "^2.25.0",
 		"ava": "^2.0.0",
 		"benchmark": "^2.1.4",
 		"codecov": "^3.3.0",
 		"del-cli": "^3.0.0",
 		"delay": "^4.2.0",
-		"eslint-config-xo-typescript": "^0.16.0",
+		"eslint-config-xo-typescript": "^0.27.0",
 		"in-range": "^2.0.0",
 		"nyc": "^14.0.0",
 		"random-int": "^2.0.0",
 		"time-span": "^3.1.0",
 		"ts-node": "^8.3.0",
-		"typescript": "3.7.2",
+		"typescript": "^3.8.3",
```

Comment:
This should stay locked. TypeScript breaks stuff all the time in minor releases.